2023-03-14 17:03:48,637 (ERROR) - 241 - django.request - Cannot assign "<django.contrib.auth.models.AnonymousUser object at 0x7f8fdc9c1bb0>": "Camions.user" must be a "User" instance.: /api/camions/
rest_framework.exceptions.APIException: Cannot assign "<django.contrib.auth.models.AnonymousUser object at 0x7f8fdc9c1bb0>": "Camions.user" must be a "User" instance.
